pkg/subctl/cmd/gather.go

```diff
@@ -183,9 +184,25 @@ func gatherConnectivity(dataType string, info *gather.Info) bool {
 func gatherDiscovery(dataType string, info *gather.Info) bool {
 	switch dataType {
 	case Logs:
-		info.Status.QueueWarningMessage("Gather ServiceDiscovery Logs not implemented yet")
+		err := gather.ServiceDiscoveryPodLogs(info)
+		if err != nil {
+			info.Status.QueueFailureMessage(fmt.Sprintf("Failed to gather all ServiceDiscovery pod logs: %s", err))
+		} else {
+			info.Status.QueueSuccessMessage("Successfully gathered all ServiceDiscovery pod logs")
```

Comment:
We don't need this - gatherPodLogs prints a success message (merge pending).

Reply:
Done